src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,47 @@
+package duke;
+
+import duke.ui.Ui;
+import duke.ui.parser.*;
```

Comment:
As per the coding standard, imported classes should always be listed explicitly.